### PR TITLE
Added (welcome, instruction) screen, (day, night, rain) mode

### PR DESCRIPTION
**Pull request by AshRahman**
added welcome screen and instruction screen

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -341,16 +375,179 @@ void DrawCity()
...
+    glColor3f(1.000, 0.980, 0.941);
+    drawstring(150.0,560.0,0.0,"PRESS 'C'");
+    glColor3f(1.000, 0.980, 0.941);
+    drawstring(300.0,520.0,0.0,"RAIN MODE");
+    glColor3f(1.000, 0.980, 0.941);
+    drawstring(150.0,520.0,0.0,"PRESS 'R'");
+    glColor3f(1.000, 0.980, 0.941);
+    drawstring(300.0,480.0,0.0,"NO RAIN");
+    glColor3f(1.000, 0.980, 0.941);
+    drawstring(150.0,480.0,0.0,"PRESS 'E'");
```
Comment (IamLizu) [inline main.cpp]: Wrong instruction, according to line 596 rain supposed to stop with 's'

File: main.cpp [hunk 2]
```diff
@@ -341,16 +375,179 @@ void DrawCity()
...
+    glColor3f(1.000, 0.980, 0.941);
+    drawstring(300.0,440.0,0.0,"TOP VIEW");
+    glColor3f(1.000, 0.980, 0.941);
+    drawstring(150.0,440.0,0.0,"PRESS 'T'");
```
Comment (IamLizu) [inline main.cpp:561]: Dont work

File: main.cpp [hunk 3]
```diff
@@ -341,16 +375,179 @@ void DrawCity()
...
+    glColor3f(1.000, 0.980, 0.941);
+    drawstring(300.0,400.0,0.0,"PLANE MOVEMENT");
+    glColor3f(1.000, 0.980, 0.941);
+    drawstring(150.0,400.0,0.0,"PRESS 'P'");
```
Comment (IamLizu) [inline main.cpp:565]: Dont work

File: main.cpp [hunk 4]
```diff
@@ -341,16 +375,179 @@ void DrawCity()
...
+    glColor3f(1.000, 0.980, 0.941);
+    drawstring(300.0,360.0,0.0,"Help");
+    glColor3f(1.000, 0.980, 0.941);
+    drawstring(150.0,360.0,0.0,"PRESS 'H'");
+    glColor3f(1.000, 0.980, 0.941);
+    drawstring(300.0,320.0,0.0,"Escape");
+    glColor3f(1.000, 0.980, 0.941);
+    drawstring(150.0,320.0,0.0,"PRESS 'ESC'");
```
Comment (IamLizu) [inline main.cpp:573]: Dont work

**Review by IamLizu — CHANGES_REQUESTED** [review]
Please remove or fix not working options also replace the wrong instruction to stop rain with right one.

File: main.cpp (hunk 1 above)
Comment (AshRahman) [reply main.cpp]: Instruction func can be changed as they are strings!so i left them for now,as all the works are not done yet

File: main.cpp (hunk 2 above)
Comment (AshRahman) [reply main.cpp:561]: Not needed

File: main.cpp (hunk 3 above)
Comment (AshRahman) [reply main.cpp:565]: No need as usual

File: main.cpp (hunk 4 above)
Comment (AshRahman) [reply main.cpp:573]: This will be needed in the final one,so it can be left here

**Review by IamLizu — APPROVED** [review]

**Pull request merged**